setup.py

```diff
@@ -18,6 +18,21 @@ def get_version(rel_path):
 with open("README.md", "r") as readme_file:
     long_description = readme_file.read()
 
+
+solver_dependencies = {
```

Comment:
Can't we load this dynamically from `Solverlookup`? I guess not seeing we also have the version requirements...
__for the future__ we could add a "get_pip_str" method to the solver-interface or add a "MIN_VERSION" attribute so this can be done automatically